scripts/script-tag.js

```diff
@@ -82,19 +82,21 @@
             preorderButton.off('click', addToCart);
             preorderButton.siblings().css('display', 'block');
 
-            const checkItemAvailablity = await isItemAvailableForOrder().then((product) => {
+            let checkItemAvailablity = await isItemAvailableForOrder().then((product) => {
                 // checking what type of tag product contains (Pre-Order / Back-order) and on the basis of that will check for metafield
                 productType = product.tags.includes('Pre-Order') ? 'Pre-Order' : product.tags.includes('Back-Order') ? 'Back-Order' : ''
 
                 // checking if continue selling is enabled for the variant or not
                 return product.variants.find((variant) => variant.id == variantId).available
             }).catch(err => err);
 
+            checkItemAvailablity = !(jQueryPreOrder("input[id='hc_inventory']").val() > 0) && (Object.entries(hc_inventory_policy).find(([key, value]) => key == variantId)[1] != "continue");
```

Comment:
I think second condition should be negated
`(Object.entries(hc_inventory_policy).find(([key, value]) => key == variantId)[1] != "continue")`
https://github.com/hotwax/shopify-preorder-app/pull/29/files#diff-3084686847e3b7fb3047950d422545acbb0ed6d13b47b91bbc0491d6602839caR95